### DISCUSSION TITLE
[Bug report] RSS/JSON Feed title setting > 'Page name' should read 'Site name'

### DISCUSSION THREAD

**Discussion by bttrx** [GetPublii/Publii#1914] [Bug Report]
### Operating system

Linux Mint 22 MATE

### Publii version

0.46.3 (build 17008)

### Issue type

Application

### Bug description

![grafik](https://github.com/user-attachments/assets/13c708d0-2fac-4d27-8b47-5b1b6e39d09a)

**Comment by michaelfluegge** [comment]
I am no pro with feeds, but you could switch to "custom feed title" and put the site name in?

**Reply by bttrx** [reply]
That's not the point. It's just a spelling mistake. If I chose **Page name** (the default), Publii writes the site's name to the feed. So it already does the right thing, but **Page name** is wrong for the drop-down menu. **Site name** would be the correct phrase.

**Reply by michaelfluegge** [reply]
Oh, I see. I misunderstood your description.
